libraries/botbuilder-dialogs-adaptive/src/adaptiveDialog.ts

```diff
@@ -325,11 +373,13 @@ export class AdaptiveDialog<O extends object = {}> extends DialogContainer<O> {
         }
     }
 
-    private async queueFirstMatch(sequence: SequenceContext, event: DialogEvent, preBubble: boolean): Promise<boolean> {
-        for (let i = 0; i < this.triggers.length; i++) {
-            const changes = await this.triggers[i].evaluate(sequence, event, preBubble);
-            if (changes && changes.length > 0) {
-                sequence.queueChanges(changes[0]);
+    private async queueFirstMatch(sequenceContext: SequenceContext, event: DialogEvent, preBubble: boolean): Promise<boolean> {
+        const selection = await this.selector.select(sequenceContext);
+        if (selection.length > 0) {
+            const evt = this.triggers[selection[0]];
+            const changes = await evt.executeAsync(sequenceContext);
```

Comment:
>Async [](start = 45, length = 5)

we don't put the Async on the end of methods in JS/TS. That's a C# pattern.

Reply:
Removed async suffix on async functions.